tests/integration-tests/tests-tlr/src/test/java/edu/kit/kastel/mcse/ardoco/tlr/tests/integration/tlrhelper/ModelElementSentenceLink.java

```diff
@@ -8,15 +8,15 @@
 /**
  * Represents a simple trace link by the id of the model and number of the sentence involved.
  */
-public record TestLink(String modelId, int sentenceNr) implements Comparable<TestLink> {
+public record ModelElementSentenceLink(String modelElementId, int sentenceNr) implements Comparable<ModelElementSentenceLink> {
```

Comment:
Was this way before, but maybe we can directly clean up:
I suggest to rename the `sentenceNr` in `sentenceNumber`